MaterialDesignThemes.Wpf/SnackbarMessageQueue.cs

```diff
@@ -381,7 +383,7 @@ private async Task ShowAsync(Snackbar snackbar, SnackbarMessageQueueItem message
             };
             snackbar.SetCurrentValue(Snackbar.MessageProperty, snackbarMessage);
             snackbar.SetCurrentValue(Snackbar.IsActiveProperty, true);
-            return new MouseNotOverManagedWaitHandle(snackbar);
+            return Tuple.Create(snackbarMessage, new MouseNotOverManagedWaitHandle(snackbar));
```

Comment:
Personally, I prefer `ValueTuple` and deconstructing instead of `Tuple`.

Reply:
Me too, but when I tried to use it, Visual Studio said...
> Predefined type 'System.ValueTuple`2' is not defined or imported

...so I decided to make a more minimal change by sticking with `System.Tuple`.

![2020-09-15_08-56-55_299_devenv](https://user-images.githubusercontent.com/34664007/93220187-ce034380-f731-11ea-8cb5-448ab29770fb.png)

Reply:
Add this to MaterialDesignThemes.Wpf.csproj
```xml
  <ItemGroup Condition="$(TargetFramework) == 'net45'">
    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
  </ItemGroup>
```

Reply:
Normally I would, but MDinXAML is using paket, and I don't know how to use paket.

Reply:
I see, I don't know either.

Reply:
I think using `System.Tuple` is fine for this PR.  A future PR can switch to using `System.ValueTuple` if that is desired.

Reply:
I got how to use paket.

Add to paket.dependencies.
```
nuget System.ValueTuple 4.5.0 framework: net45
```
Add to MaterialDesignThemes.Wpf\paket.references,
```
System.ValueTuple
```
Then run this command.
```
.paket\paket.exe install
```